nservicebus/messaging/publish-subscribe/publish-handle-event.md

```diff
@@ -39,9 +39,10 @@ snippet:DefiningEventsAs
 
 An event can be handled by use of the `IHandleMessages` interface on any [Handler](/nservicebus/handlers) or [Saga](/nservicebus/sagas).
 
-
 ## Publishing an event
 
+NOTE: In Version 6, the IBus interface has been deprecated and removed. Use the `Publish` methods on the `IMessageHandlerContext` interface instead. For more information, please consult the [upgrade guide](/nservicebus/upgrades/5to6.md).
```

Comment:
This should be the definative source on how to do a publiish in any version. It should not refer to the upgrade guide